src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -319,6 +321,61 @@ def fuzz(self,
                                 artifact_prefix, additional_args)
 
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+
+  def __init__(self, executable_path, default_args=None):
+    if (not environment.get_value('FUCHSIA_PKEY_PATH')) or (
+        not environment.get_value('FUCHSIA_PORTNUM')):
+      logs.log_error('FUCHSIA_PKEY_PATH and/or FUCHSIA_PORTNUM was not set')
+      return
+    self.ssh_root = [
+        'ssh', '-i',
+        str(environment.get_value('FUCHSIA_PKEY_PATH')), '-o',
+        'StrictHostKeyChecking no', 'localhost', '-o',
+        'UserKnownHostsFile=/dev/null', '-p',
+        str(environment.get_value('FUCHSIA_PORTNUM'))
```

Comment:
same here.

Reply:
python interprets a portnumber as an int, rather than a str, so it makes sense to coerce it here.  unless you see another way?

Reply:
ah, I thought you were doing it to make sure it's not None. ignore!